### ISSUE TITLE
Code Blocks are horribly broken when in a reply

### ISSUE THREAD

**Issue by ghost** [vector-im/element-android#2247]
**Describe the bug**
Code blocks `<pre><code></code></pre>` are horribly broken for android users. Instead of interpreting the html correctly, the client seems to try to parse the contents as partial markdown of sorts? This only seems to happen to RichEmbed replies, not normal posts.

**To Reproduce**
```json
{
    "msgtype": "m.notice",
    "body": "This doesn't really matter for recreating the issue",
    "format": "org.matrix.custom.html",
    "formatted_body": "<mx-reply><blockquote><a href=\"https://matrix.to/#/!nowhere:example.com/$nowhere\">In reply to</a><a href=\"https://matrix.to/#/@noone:example.com\">@noone:example.com</a><br />Some text</blockquote></mx-reply><pre><code class=\"language-plain\">EXAMPLE(1)                     Element Android                         HELP(1)\n\n<b>NAME</b>\n       example - example page\n\n<b>DESCRIPTION</b>\n       <b>example</b> is a manual page to recreate failures in the Element Android message renderer.\n\n<b>EXAMPLE</b>\n       <b>example</b> list\n           Lorus Ipsum\n\n       <b>__example__</b> list\n           Lorus Ipsum\n\n       <b>[example](https://example.com)</b> list\n           Lorus Ipsum\n\n       <b>**example**</b> list\n           Lorus Ipsum\n\n1.0.0                             2020-10-11                        EXAMPLE(1)\n</code></pre>\n"
}
```

**Expected behavior**
Element should just render the html normally. Codeblocks in general are broken in Element Android (and Riot.im Android) but this breakage is exceptional.

**Screenshots**
![element](https://user-images.githubusercontent.com/16590431/95753642-cdf75480-0c99-11eb-8899-f027b6ac2a8c.jpg)

**Smartphone (please complete the following information):**
 - Device: Samsung S8
 - OS: Android 9

**Additional context**
 - App version and store: 1.0.8 Play

**Comment by kakolisgay** [comment]
This bug still exists as of 1.0.16

**Comment by HarHarLinks** [comment]
Some related things appear to have been fixed, e.g. #3576
Is this still an issue?

**Comment by cloudrac3r** [comment]
This is fixed by my #6357 which is in release 1.4.26. Try updating.
